### PR TITLE
refac: build events overview page

### PR DESCRIPTION
**Pull request by Otaiki1**
Build events overview page -draft PR

### CODE REVIEW

File: src/pages/DashboardPage.jsx [hunk 1]
```diff
@@ -1,13 +1,38 @@
...
-import EventsList from "../components/EventsList";
-
+import addTeam from "../assets/add-team.svg";
+import share from "../assets/share-08.svg";
+import Button from "../components/Button";
+import EventCard from "../components/EventCard";
+import Footer from "../components/Footer";
+import HostsCard from "../components/HostsCard";
+import VisibilityCard from "../components/VisibilityCard";
 function DashboardPage() {
```
Comment (mubarak23) [inline src/pages/DashboardPage.jsx:8]: link this page to a route

**Review by mubarak23 — CHANGES_REQUESTED** [review]
pull from main and fix the merge conflict

File: src/pages/DashboardPage.jsx (hunk 1 above)
Comment (mubarak23) [reply src/pages/DashboardPage.jsx:8]: "/home" is not throwing blank page,

Comment (Otaiki1) [reply src/pages/DashboardPage.jsx:8]: Current route is "/app/home"
changing to /home would affect sidebar and layout settings

**Pull request merged**